### DISCUSSION TITLE
growth animation

### DISCUSSION THREAD

**Discussion by etma101** [inca/blender-differential-growth#11] [Ideas]
hi, i loved your addon. at each step it grows like its from nature. is it possible to save it as animation?. thank you.

**Comment by bejucoo** [comment]
I think it may be possible with Shape Keys? [Like in this video](https://www.youtube.com/watch?v=B11XBZmIIZc). It's not the shortest way but it's a start. Maybe the addon can be modified to add a shape key in every diff growth step.

**Reply by inca** [reply]
Interesting idea! But no I don't think Shape Keys are the right tool for the job: 

<img width="734" alt="image" src="https://user-images.githubusercontent.com/118196/233451183-2b63b258-5de0-49e6-886d-bbe5beea0db8.png">

**Reply by bejucoo** [reply]
Right. I think a variable number of vertices in shape keys could be a cool feature but i guess it may not be very easy to implement.

I found two options:

- Apply a Subdivision Surface modifier to the resulting mesh (optional), sort the vertices AND the faces to define a direction,  add a Build modifier. Is not the growth per se but kinda works:

https://user-images.githubusercontent.com/19770567/233491730-231faedd-2660-4d39-b0ab-e590b7f9f89e.mp4

-  Save every step of the growth as a mesh (Shift-D), build a GeoNodes setup (like [this one](https://www.youtube.com/watch?v=9ubfb2K6mL0)) to morph between each mesh. But with a lot of steps is gonna be messy.
